pages/your_tasks/data_discoverability.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Data Discoverability
+description: How to make data discoverable
+contributors: [Aina Jené Cortada, Laura Portell Silva]
```

Comment:
Do you have more info about Aina Jené Cortada for in the CONTRIBUTORS file?

Reply:
sure! I thought she was included already

Reply:
done!